### DISCUSSION TITLE
Sophos incompatibility

### DISCUSSION THREAD

**Discussion by JVanAlsenoy** [iterate-ch/cyberduck#15290] [Q&A] [answered]
Hi - I read in the documentation that "Sophos Anti-Virus On-access Scanning" is incompatible with Cyberduck.  I would like to use Cyberduck as a cloud-backup for local files.  Would it be sufficient to turn off the on access-scanning during the synchronisation session? (I am very new to the domain, so the question might be badly phrased).  Is there another MacOS anti-virus product that is not incompatible with Cyberduck? Any pointers or info from anyone?
Thank you for your help --
Jacques

**Comment by dkocher** [comment, marked as answer]
You are referring to _Mountain Duck_. Cyberduck is not affected by the issues caused by Sophos. The software causes two issues
* Missing icon overlays in Finder [^1]
* Performance degradation when the anti virus scanner is reading folder contents and files on the mounted volume.

[^1]: https://support.sophos.com/support/s/article/KB-000038112?language=en_US

**Reply by JVanAlsenoy** [reply]
Thank you for the clarification!  I was confused because of the statement at https://docs.cyberduck.io/cyberduck/faq/#incompatibilities
Getting closer to testing!!
